docs/js/controls-toggles.js

```diff
@@ -0,0 +1,12 @@
+export default 
+  window.requestAnimationFrame(() => {
+    let slider = document.querySelector("#slider1");
+    let counter = document.querySelector(".a-slider--value");
+
+    slider.addEventListener("input", function (){
+      let sliderValue = slider.value;
```

Comment:
We can use const instead of let here and on lines 3 and 4.

```suggestion
      const sliderValue = slider.value;
```

Reply:
d762264